### PR TITLE
Update fastify to v3

### PR DESCRIPTION
**Pull request by frikille**
* Updated fastify to v3
* Replaced greenkeeper config with dependabot
* Moved CI from Travis to Github actions

This PR unblocks fastify/fastify#2217

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "daily"
+    ignored_updates:
+      - match:
+          dependency_name: "tap"
```
Comment (jsumners) [inline .dependabot/config.yml:8]: We can go to latest `tap` now that we don't have to support older Node versions.

Comment (frikille) [reply .dependabot/config.yml:8]: Right, will remove it

File: package.json [hunk 2]
```diff
@@ -29,15 +29,15 @@
...
     "pre-commit": "^1.2.2",
     "snazzy": "^8.0.0",
     "standard": "^14.0.0",
     "tap": "^12.6.6"
   },
   "dependencies": {
     "abstract-cache": "^1.0.1",
-    "fastify-plugin": "^1.2.1",
+    "fastify-plugin": "^2.0.0",
     "uid-safe": "^2.1.5"
```
Comment (jsumners) [inline package.json:41]: Check for new versions of these.

**Comment by mcollina** [comment]
@frikille can you please update the dependencies as well?

File: package.json (hunk 2 above)
Comment (frikille) [reply package.json]: `tap` was the only outdated dependency, updated to the latest

**Pull request merged**